engine/syncer.go

```diff
@@ -17,8 +17,8 @@ const (
 	SyncItemOrderbook
 	SyncItemTrade
 
-	defaultSyncerWorkers = 30
-	defaultSyncerTimeout = time.Second * 15
+	DefaultSyncerWorkers = 15
```

Comment:
Yeah, in the current configuration it's overkill since most exchanges will use batching (1 request compared to 10+) and store a local cache. Timing a whole sync with default settings with 150 pairs takes 17~ seconds

```
[DEBUG] | 21/11/2019 13:08:48 | BTSE ticker sync complete BTC-USD [150/150].
[DEBUG] | 21/11/2019 13:08:48 | Exchange CurrencyPairSyncer initial sync is complete.
[DEBUG] | 21/11/2019 13:08:48 | Exchange CurrencyPairSyncer initiial sync took 17.389668s [150 sync items].
```

With websocket enabled for applicable exchanges, the requirement becomes less since we're getting frequent updates (for most of them, exception being btc markets). I've made this a configurable value in case a persons requirements change and can also revisit this in syncer 2.0